iotix/parts/footer.html

```diff
@@ -1 +1,51 @@
-<!-- wp:pattern {"slug":"iotix/footer"} /-->
\ No newline at end of file
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"5vh","right":"5vw","bottom":"10vh","left":"5vw"}}},"layout":{"type":"constrained"}} -->
```

Comment:
Looks like this footer content moved from a pattern to this template part.

I think the content needs to go back to the pattern.

Reply:
Done in [a2ddcbe](https://github.com/Automattic/themes/pull/7162/commits/a2ddcbe6724c0bb4d4570c528d027c0efd82ed91)